frontend-web/webclient/app/Accounting/Allocations.tsx

```diff
@@ -525,20 +530,39 @@ function stateReducer(state: State, action: UIAction): State {
                     ({balance: wallet.maxUsable, category: wallet.paysFor})
                 );
 
+                const retired = wallets.map(wallet => {
+                    let totalRetired = 0
+                    wallet.allocationGroups.forEach(group =>
+                        group?.group?.allocations.forEach(alloc =>
+                            totalRetired += alloc.retiredUsage ?? 0
+                        )
+                    )
+                    return ({balance: totalRetired, category: wallet.paysFor})
+                })
+
                 const combinedQuotas = Accounting.combineBalances(quotaBalances);
                 const combinedUsage = Accounting.combineBalances(usageBalances);
                 const combineMaxUsable = Accounting.combineBalances(maxUsableBalances);
+                const combineRetire = Accounting.combineBalances(retired)
+
+                const combineShouldUseRetired = wallets.map(wallet =>
+                    wallet.paysFor.accountingFrequency === "ONCE"
+                )
 
                 for (let i = 0; i < combinedQuotas.length; i++) {
                     const usage = combinedUsage[i];
                     const quota = combinedQuotas[i];
                     const maxUsable = combineMaxUsable[i]
+                    const retired = combineRetire[i]
+                    const shouldUseRetired = combineShouldUseRetired[i]
 
                     entry.usageAndQuota.push({
                         usage: usage.normalizedBalance,
                         quota: quota.normalizedBalance,
                         unit: usage.unit,
-                        maxUsable: maxUsable.normalizedBalance
+                        maxUsable: maxUsable.normalizedBalance,
+                        retired: retired.normalizedBalance,
```

Comment:
Might make more sense to call it "retiredAmount" as it otherwise sounds like a boolean value